README.md

```diff
@@ -1,18 +1,20 @@
 # Mentorship Client
 
-Cross-platform client for [Systers&#x27; mentorship system](https://github.com/systers/mentorship-backend),
-written in [Flutter](https://flutter.dev/).
+Cross-platform client for the [AnitaB.org's](https://anitab.org/) [Mentorship System](https://github.com/anitab-org/mentorship-backend),
```

Comment:
```suggestion
[Mentorship System](https://github.com/anitab-org/mentorship-backend) is an application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period. This is the flutter client for this project.
```
I would avoid having AnitaB.org's because there are other platforms that AnitaB.org uses, and this is still in development and part of the open source community for now.